femwell/maxwell/waveguide.py

```diff
@@ -329,8 +329,9 @@ def plot_component(
 
         if component == "x" or component == "y":
             plot_basis = mfield_t_basis.with_element(ElementVector(ElementDG(ElementTriP1())))
-            mfield_xy = plot_basis.project(mfield_t_basis.interpolate(conv_func(mfield_t)))
+            mfield_xy = plot_basis.project(mfield_t_basis.interpolate(mfield_t))
```

Comment:
```suggestion
            mfield_xy = plot_basis.project(conv_func(mfield_t_basis.interpolate(mfield_t)))
```
would this solve the problem?

Reply:
Thanks for the the quick reply!

I update the code according to your suggestion and it seems to work. 
Let me know if something else is needed.

Reply:
perfect, thanks!